src/main/java/sqlline/Commands.java

```diff
@@ -927,6 +937,21 @@ private void execute(String line, boolean call, DispatchCallback callback) {
 
       try {
         long start = System.currentTimeMillis();
+        String pattern = sqlLine.getOpts().getConfirmPattern();
+        if (Objects.equals(BuiltInProperty.CONFIRM_PATTERN
+                .defaultValue(), pattern)) {
+          pattern = sqlLine.loc("default-confirm-pattern");
+        }
+        if (Pattern.compile(pattern).matcher(sql).matches()
+                && sqlLine.getOpts().getConfirm()) {
+          String question = sqlLine.loc("really-perform-action");
+          if (!sqlLine.getLineReader().readLine(question)
```

Comment:
@julianhyde This was referred from the existing implementation of dropall command which doesn't seem to work after the recent changes https://github.com/julianhyde/sqlline/commit/f003fdb1ef6f3119f508c47d651c6cede46ce6c8. Could you help me understand what is missing here?